command/cp.go

```diff
@@ -80,6 +80,10 @@ var copyCommandFlags = []cli.Flag{
 		Name:  "parents",
 		Usage: "create same directory structure of source, starting from the first wildcard",
 	},
+	&cli.BoolFlag{
+		Name:  "no-follow-symlinks",
+		Usage: "do not follow symbolic links (s5cmd follows symbolic links by default)",
```

Comment:
cli package prints the default value. we may not need `(s5cmd follows symbolic links by default)`